core-plugins/src/main/java/io/cdap/plugin/common/FileSetUtil.java

```diff
@@ -68,12 +65,7 @@ public static void configureORCFileSet(String configuredSchema, FileSetPropertie
     String hiveSchema = parseHiveSchema(lowerCaseSchema, configuredSchema);
     hiveSchema = hiveSchema.substring(1, hiveSchema.length() - 1);
```

Comment:
hiveSchema is not used anymore. Also `.build()` in the end has little sense. Overall this method does same as previous - adds a single property

Reply:
Noted. Hive related changes will be handled in a separate PR.